src/App.jsx

```diff
@@ -1,14 +1,32 @@
 import Home from "./pages/Home";
+import TopicSelect from "./pages/TopicSelect";
 import { Routes, Route } from "react-router-dom";
+import { useState } from "react";
+import Article from "./pages/Article";
 
 function App() {
-	return (
-		<div className="App">
-			<Routes>
-				<Route path="/" element={<Home />} />
-			</Routes>
-		</div>
-	);
+  const [topic, setTopic] = useState({});
+  const [topics, setTopics] = useState([]);
```

Comment:
It looks like only the TopicSelect element uses these pieces of state - why do they need to be in App?